launch-game.sh

```diff
@@ -25,7 +25,16 @@ mono OpenRA.Game.exe Engine.LaunchPath="$MODLAUNCHER" $MODARG "$@"
 
 # Show a crash dialog if something went wrong
 if [ $? != 0 ] && [ $? != 1 ]; then
-	ERROR_MESSAGE="OpenRA has encountered a fatal error.\nPlease refer to the crash logs and FAQ for more information.\n\nLog files are located in ~/.openra/Logs\nThe FAQ is available at http://wiki.openra.net/FAQ"
+	if [[ "$OSTYPE" == "darwin"* ]]; then
+		LOGS="~/Library/Application Support/OpenRA/Logs/"
+	else
+		LOGS="~/.openra/Logs"
+		if [ ! -d "$LOGS" ]; then
+			LOGS="~/.config/openra/Logs"
```

Comment:
```suggestion
			if [ ! -d "$XDG_CONFIG_HOME" ]; then
				LOGS="~/.config/openra/Logs"
			else
				LOGS="$XDG_CONFIG_HOME/openra/Logs"
```


If $XDG_CONFIG_HOME is either not set or empty, a default equal to $HOME/.config should be used.

http://standards.freedesktop.org/basedir-spec/latest

Reply:
This also matches the logic implemented inside `Platform.cs`.

Reply:
Thanks, updated.